numeral.js

```diff
@@ -54,10 +54,15 @@
           boundedPrecision = minDecimals;
         }
 
+        // Number.toFixed requires it's argument be between 0 and 20, inclusive.
+        // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed
+        boundedPrecision = Math.max(Math.min(boundedPrecision, 20), 0);
```

Comment:
I don't think so. `toFixed` is called in 3 places:

When there are optional decimal parts (`0,0.0000[000]`):
```
d = toFixed(value, (precision[0].length + precision[1].length), roundingFunction, precision[1].length);
```
Here precision is an array of strings, so the lengths are always non-negative, and `precision[1].length <= precision[0].length + precision[1].length`, so `minDecimals <= maxDecimals`


When there are no optional parts, but there is a decimal part (`0,0.0000`):
```
d = toFixed(value, precision.length, roundingFunction);
```
Here `precision` is a string. This sets `minDecimals` to `0`, and it must be that `maxDecimals >= 0`

And when there is no decimal part (`0,0`):
```
w = toFixed(value, null, roundingFunction);
```
Here `boundedPrecision` is set to `minDecimals` which will be `0`.